cddl/profile.cddl

```diff
@@ -1,3 +1,3 @@
 profile-claim = (
-    profile => tstr
+    profile => tstr / oid
```

Comment:
CDDL nits: I think we need to import the definition of oid from https://datatracker.ietf.org/doc/draft-ietf-cbor-tags-oid/, i.e.: `oid = #6.111(bstr)`

Also, if we are requiring an URL, maybe better letting the type system help out: `uri = #6.32(tstr)`?